Makefile

```diff
@@ -107,15 +107,21 @@ test-test262-update-whitelist:
 clone-license:
 	./scripts/clone-license.sh
 
-publish:
-	git pull --rebase
+prepublish-build:
 	make clean-lib
 	rm -rf packages/babel-runtime/helpers
 	rm -rf packages/babel-runtime-corejs2/helpers
 	rm -rf packages/babel-runtime-corejs2/core-js
 	BABEL_ENV=production make build-dist
-	make test
 	make clone-license
+
+prepublish:
+	git pull --rebase
+	make prepublish-build
+	make test
+
+publish:
```

Comment:
you can use `publish: prepublish` here, it's equivalent to `make prepublish` the line under.